Modules/MainFrame/Frames.lua

```diff
@@ -773,11 +852,13 @@ end
 function Arenalogs.MAIN_FRAME.FRAMES:UpdateMatchHistory()
     local matchHistoryTab = Arenalogs.MAIN_FRAME.frame.content.matchHistoryTab
     local matchHistoryList = matchHistoryTab.content.matchHistoryList
+    local matchHistoryListCharacter = matchHistoryTab.content.matchHistoryList

```

Comment:
this is the same value as the line above (The GGUI.FrameList Widget for the match history)